src/Stratis.Bitcoin.Features.MemoryPool/MempoolOrphans.cs

```diff
@@ -265,14 +274,16 @@ public class OrphanTx
         /// <param name="from">Source node for transaction.</param>
         /// <param name="tx">Transaction to add.</param>
         /// <returns>Whether the transaction was added to orphans.</returns>
-        public async Task<bool> ProcessesOrphansMissingInputsAsync(INetworkPeer from, Transaction tx)
+        public bool ProcessesOrphansMissingInputsAsync(INetworkPeer from, Transaction tx)
```

Comment:
not async anymore

Reply:
No it was async only for the mempool lock, other then that nothing else needed asyncness

Reply:
That was not question. Remove the suffix

Reply:
Done